packages/collector/test/tracing/database/redis/test.js

```diff
@@ -85,6 +87,52 @@ const globalAgent = require('../../../globalAgent');
                 });
               });
             });
+            // In v5, Redis moved “Isolation Pool” into RedisClientPool.
+            // see: //github.com/redis/node-redis/blob/master/docs/pool.md
+            if (redisVersion === 'latest') {
+              mochaSuiteFn('When connected via clientpool', function () {
+                globalAgent.setUpCleanUpHooks();
+                let controls;
+
+                before(async () => {
+                  controls = new ProcessControls({
+                    useGlobalAgent: true,
+                    appPath:
+                      redisVersion === legacyVersion
+                        ? path.join(__dirname, 'legacyApp.js')
+                        : path.join(__dirname, 'app.js'),
+                    env: {
+                      REDIS_VERSION: redisVersion,
+                      REDIS_PKG: redisPkg,
+                      REDIS_POOL: true
+                    }
+                  });
+
+                  await controls.startAndWaitForAgentConnection(5000, Date.now() + 1000 * 60 * 5);
+                });
+
+                beforeEach(async () => {
+                  await agentControls.clearReceivedTraceData();
+                });
+
+                before(async () => {
+                  await controls.sendRequest({
+                    method: 'POST',
+                    path: '/clearkeys'
+                  });
+                });
+
+                after(async () => {
+                  await controls.stop();
+                });
+
+                afterEach(async () => {
+                  await controls.clearIpcMessages();
+                });
+
+                it('should trace blocking commands', () => testBlockingCommand(controls, setupType));
```

Comment:
I don't understand this test. Blocking commands? 

>                    method: 'POST',
                    path: '/clearkeys'